pkgs/data/fonts/garamond-libre/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchzip }:
+
+let
+  version = "1.4";
+in fetchzip {
+  name = "garamond-libre-${version}";
```

Comment:
```suggestion
fetchzip {
  pname = "garamond-libre";
  version = "1.4";
```

Reply:
There are `${version}` occurence in the url, I have to use `let...in`.

Reply:
Fixed